packages/react-dev-utils/FlowTypecheckPlugin.js

```diff
@@ -109,6 +108,7 @@ function getFlowVersion(options) {
 }
 
 function initializeFlow(projectPath, flowconfig, otherFlowTypedDefs) {
+  const flowTypedPath = path.join(projectPath, 'node_modules', '.bin', 'flow-typed');
```

Comment:
I honestly never use packages like this so I'll have to be a bit smarter about this...